app/views/reports/regions/show.html.erb

```diff
@@ -224,6 +224,87 @@
     </div>
   </div>
 </div>
+<div class="mt-8px mx-0px mb-16px p-20px bg-white br-4px bs-small d-lg-flex fd-lg-column justify-lg-between h-lg-full w-lg-full mt-lg-0">
```

Comment:
I would recommend extracting this to a partial to keep `show` from getting any longer.